README.md

```diff
@@ -1,3 +1,7 @@
+As of November 2021, this project is not longer maintained because it has moved to [addons-linter](https://github.com/mozilla/addons-linter), see: https://github.com/mozilla/addons-linter/issues/869.
```

Comment:
Nit: Could we make this a bit more prominent, like a second level headline or so?

Reply:
Like this?
![Screenshot 2021-11-09 at 12 18 56](https://user-images.githubusercontent.com/217628/140914870-d71ac186-383e-4514-a6e3-94ff55e5225f.png)

Reply:
Looks good!